sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -967,6 +967,138 @@ test_suite(
     ],
 )
 
+otp_json(
+    name = "shutdown_alert_owner_sw_cfg",
+    partitions = [
+        otp_partition(
+            name = "OWNER_SW_CFG",
+            items = {
+                "OWNER_SW_CFG_DIGEST": "0x0",
+                # Enable bootstrap. See `hardened_bool_t` in
+                # sw/device/lib/base/hardened.h.
+                "OWNER_SW_CFG_ROM_BOOTSTRAP_EN": "0x739",
+                # Set to 0x739 to use the ROM_EXT hash measurement as the key
+                # manager attestation binding value.
+                "OWNER_SW_CFG_ROM_KEYMGR_ROM_EXT_MEAS_EN": "0x0",
+                # Report errors without any redaction.
+                "OWNER_SW_CFG_ROM_ERROR_REPORTING": "0xe2290aa5",
+                # Set the enables to kAlertEnableEnabled.
+                # See `alert_enable_t` in
+                # sw/device/silicon_creator/lib/drivers/alert.h
+                "OWNER_SW_CFG_ROM_ALERT_CLASS_EN": "0x07070707",
+                #"OWNER_SW_CFG_ROM_ALERT_CLASS_EN": "0xa9a9a9a9",
+                # Set the esclation policies to kAlertEscalateNone.
+                # See `alert_escalate_t` in
+                # sw/device/silicon_creator/lib/drivers/alert.h
+                "OWNER_SW_CFG_ROM_ALERT_ESCALATION": "0x76767676",
+                #"OWNER_SW_CFG_ROM_ALERT_ESCALATION": "0xd1d1d1d1",
+                # Set the classifiactions to kAlertClassX.
+                # See `alert_class_t` in
+                # sw/device/silicon_creator/lib/drivers/alert.h
+                #"OWNER_SW_CFG_ROM_ALERT_CLASSIFICATION": ["0x94949494"] * 80,
+                "OWNER_SW_CFG_ROM_ALERT_CLASSIFICATION": ["0xeeeeeeee"] * 80,
+                # Set the classifiactions to kAlertClassX. See `alert_class_t` in
+                # sw/device/silicon_creator/lib/drivers/alert.h
+                #"OWNER_SW_CFG_ROM_LOCAL_ALERT_CLASSIFICATION": ["0x94949494"] * 16,
+                "OWNER_SW_CFG_ROM_LOCAL_ALERT_CLASSIFICATION": ["0x94949494"] * 16,
+                # Set the alert accumulation thresholds to 0 per class.
+                "OWNER_SW_CFG_ROM_ALERT_ACCUM_THRESH": ["0x00000000"] * 4,
+                # Set the alert timeout cycles to 0 per class.
+                "OWNER_SW_CFG_ROM_ALERT_TIMEOUT_CYCLES": ["0x00000000"] * 4,
+                # Set the alert phase cycles to 0,10,10,0xFFFFFFFF for classes
+                # A and B, and to all zeros for classes C and D.
+                "OWNER_SW_CFG_ROM_ALERT_PHASE_CYCLES": [
+                    "0x0",
+                    "0xa",
+                    "0xa",
+                    "0xFFFFFFFF",
+                    "0x0",
+                    "0xa",
+                    "0xa",
+                    "0xFFFFFFFF",
+                    "0x0",
+                    "0x0",
+                    "0x0",
+                    "0x0",
+                    "0x0",
+                    "0x0",
+                    "0x0",
+                    "0x0",
+                ],
```

Comment:
```suggestion
                # Set the alert phase cycles to 0 for class A and to 0xffffffff for other classes.
                "OWNER_SW_CFG_ROM_ALERT_PHASE_CYCLES": ["0x0"] * 4  + ["0xffffffff"] * 28,
```

Reply:
Should this be `["0x0"] * 4  + ["0xffffffff"] * 12,`? It's my understanding that there are `PARAM_N_PHASES * PARAM_N_CLASSES = 4 * 4 = 16` fields here.

Reply:
Yeah, I think you're right.